website/content/docs/commands/operator/raft.mdx

```diff
@@ -184,3 +184,112 @@ Usage: vault operator raft snapshot restore <snapshot_file>
 
 	  $ vault operator raft snapshot restore raft.snap
 ```
+
+## autopilot
+
+This command groups subcommands for operators interacting with the autopilot
+functionality of the integrated Raft storage backend. There are 3 subcommands
+supported: `get-config`, `set-config` and `state`.
+
+```text
+Usage: vault operator raft autopilot <subcommand> [options] [args]
+
+This command groups subcommands for operators interacting with the autopilot
+functionality of the integrated Raft storage backend.
+
+Subcommands:
+    get-config    Returns the configuration of the autopilot subsystem under integrated storage
+    set-config    Modify the configuration of the autopilot subsystem under integrated storage
+    state         Displays the state of the raft cluster under integrated storage as seen by autopilot
+```
+
+### autopilot state
+
+Displays the state of the raft cluster under integrated storage as seen by autopilot.
```

Comment:
As this just repeats the usage information, I think it would be better to explain what we are seeing in the output.

I suggested name, but is that nodeID?

I don't actually know if we have a link where non-voters are described. If not, we should add somewhere? Concepts page?

```suggestion
Displays the state of the raft cluster under integrated storage as seen by autopilot. It shows whether autopilot thinks the cluster is healthy or not, and how many nodes could fail before the cluster becomes unhealthy ("Failure Tolerance").

State includes a list of all servers by name and IP address. Last Index can indicate how close the state on each node is to the leader.

A node can have a status of "leader", "voter", and "non-voter". Non-voters are described in <link>
```